### PR TITLE
Enhance REAMDE.md

### PR DESCRIPTION
**Pull request by breedx-splk**
Adds a bit of docs around render detection and storage buffering.

### CODE REVIEW

**Comment by breedx-splk** [comment]
@theletterf FYI

**Comment by jkwatson** [comment]
Should we be updating the README before these features are released? Is it time for a release?

File: README.md [hunk 1]
```diff
@@ -245,7 +247,9 @@ when initializing your instance of the SplunkRum API:
   Set/change the default polling interval for slow/frozen render detection.
   Default is 1000ms. Value must be positive. 
 - `disableSlowRenderingDetection()` :
-  Disable the detection of slow frame renders (default is enabled). 
+  Disable the detection of slow frame renders (default is enabled).
```
Comment (theletterf) [inline README.md:250]:
```suggestion
  Disables the detection of slow frame renders. Enabled by default.
```

File: README.md [hunk 2]
```diff
@@ -245,7 +247,9 @@ when initializing your instance of the SplunkRum API:
...
+- `enableDiskBuffering()` : 
+  Enables the storage-based buffering of telemetry.
```
Comment (theletterf) [inline README.md:252]: @jkwatson @breedx-splk Sounds like this might come in handy in certain scenarios. What are those?

File: README.md [hunk 3]
```diff
@@ -278,6 +282,21 @@ when initializing your instance of the SplunkRum API:
   okhttp `Call.Factory` wrapper that can be applied to your client. See
   the `createRumOkHttpCallFactory(OkHttpClient)` for details.
 
+#### Slow / Frozen Renders
```
Comment (theletterf) [inline README.md:285]:
```suggestion
#### Detection of slow or frozen renders
```

File: README.md [hunk 4]
```diff
@@ -278,6 +282,21 @@ when initializing your instance of the SplunkRum API:
...
+
+By default, Splunk RUM will detect and report slow and frozen screen renders.
+To disable this feature, call `.disableSlowRenderingDetection()` on the 
+`Config.Builder`.
```
Comment (theletterf) [inline README.md:289]:
```suggestion
By default, Splunk RUM detects and reports slow or frozen screen renders.
To disable this feature, call `.disableSlowRenderingDetection()` on the 
`Config.Builder`.
```

Comment (theletterf) [reply README.md:289]: Why would someone want to disable this feature?

File: README.md [hunk 5]
```diff
@@ -278,6 +282,21 @@ when initializing your instance of the SplunkRum API:
...
+
+The Splunk RUM definitions of "slow" and "frozen" are in line with the
+[Android Vitals definitions](https://developer.android.com/topic/performance/vitals/frozen):
+
+| category | speed   | spanName      | span attr |
+|----------|---------|---------------|-----------|
+| slow     | > 16ms  | slowRenders   | count     | 
+| frozen   | > 700ms | frozenRenders | count     |
```
Comment (theletterf) [inline README.md:297]:
```suggestion
Splunk RUM defines renders as slow or frozen following the [Android Vitals definitions](https://developer.android.com/topic/performance/vitals/frozen):

| Category | Speed   | spanName      | Attribute   |
|----------|---------|---------------|-----------|
| Slow     | >16ms  | slowRenders   | count     | 
| Frozen   | >700ms | frozenRenders | count     |
```

**Review by theletterf — CHANGES_REQUESTED** [review]
@breedx-splk When is this due for release?

**Comment by mateuszrzeszutek** [comment]
> Should we be updating the README before these features are released? 

In other repos (splunk-otel-java) we're using the `main` branch as a "development" branch; if you need docs for a particular release you have to look up tags. I think it makes sense to follow that pattern here as well.

> Is it time for a release?

I think it's high time for a release -- let's merge everything that's still open first, then we can do the release. It's been some time since the last one...

**Comment by theletterf** [comment]
@mateuszrzeszutek I'll prepare the docs update then. Anything else to make it to the release?

File: README.md (hunk 2 above)
Comment (mateuszrzeszutek) [reply README.md:252]: For example, when you have an app that may work offline for extended periods of time (like a GPS trail tracker)

Comment (theletterf) [reply README.md:252]:
```suggestion
  Enables the storage-based buffering of telemetry. This setting is useful when instrumenting applications that might work offline for extended periods of time.
```

**Comment by mateuszrzeszutek** [comment]
At the very least I'd like #226 and #255 to get in.

File: README.md [hunk 6]
```diff
@@ -245,7 +247,9 @@ when initializing your instance of the SplunkRum API:
   Set/change the default polling interval for slow/frozen render detection.
   Default is 1000ms. Value must be positive. 
 - `disableSlowRenderingDetection()` :
-  Disable the detection of slow frame renders (default is enabled). 
+  Disable the detection of slow frame renders (default is enabled).
+- `enableDiskBuffering()` :
```
Comment (mateuszrzeszutek) [inline README.md:251]: @breedx-splk I've changed some of these names in #255 to match the other properties, please take a look at that

File: README.md (hunk 2 above)
Comment (breedx-splk) [reply README.md:252]: Eventually we want this to be the default....but for now it's opt-in because it's new.

**Comment by breedx-splk** [comment]
> @breedx-splk When is this due for release?

snapshot builds are released with every merge to the main branch.
https://oss.sonatype.org/content/repositories/snapshots/com/splunk/splunk-otel-android/0.13.0-SNAPSHOT/

We can cut a minor release this week, it seems like people want it.

File: README.md (hunk 6 above)
Comment (breedx-splk) [reply README.md:251]: Cool, I think I've got it updated to match.

**Review by theletterf — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -278,6 +283,20 @@ when initializing your instance of the SplunkRum API:
   okhttp `Call.Factory` wrapper that can be applied to your client. See
   the `createRumOkHttpCallFactory(OkHttpClient)` for details.
 
+#### Detection of slow or frozen renders
+
+By default, Splunk RUM detects and reports slow or frozen screen renders.
+To disable this feature, call `.disableSlowRenderingDetection()` on the
```
Comment (mateuszrzeszutek) [inline README.md:289]:
```suggestion
To disable this feature, call `.slowRenderingDetectionEnabled(false)` on the 
```

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**